src/styles/shared.tsx

```diff
@@ -152,13 +152,13 @@ const SideBarMenu = ({sideBarOpen, toggleMenu}: SideBarMenuProps) => {
   const userName = useAppSelector(state => state.currentUser.first_name);
   const dispatch = useAppDispatch();
   const history = useHistory();
-  const { logOut, setCurrentUser, updateCurrentGroup, setCurrentUserId, purgePaymentsList } = allActions;
+  const { logOut, setCurrentUser, updateCurrentGroup, purgePaymentsList } = allActions;
   const dimmerRef = useRef(null)
 
   const handleLogOutClick = () => {
     dispatch(setCurrentUser({}));
     dispatch(updateCurrentGroup(0));
-    dispatch(setCurrentUserId(0));
+    // dispatch(setCurrentUserId(0));
```

Comment:
yes, i believe setCurrentUserId is a deprecated action, as we have been removing redundant states out of existence. This is okay to delete!